canonical-web-and-design/charmhub.io
# Update featured charms

## Done
Update featured charms
Update featured charms count

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://localhost:8045
- Check the featured charms match [the copy doc](https://docs.google.com/spreadsheets/d/1BTvg-NWD-yZjzN2eg-5NbTYZMXhrb5mDqqY63UxYJok/edit#gid=577595503) filtered by column K
